src/index.html

```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="page__month month month--length--31">
```

Comment:
Better name `calendar` instead of `month`.

Follow BEM naming convention for complex modifiers: block-name--modifier-name--modifier-value
```suggestion
    <div class="page__calendar calendar  calendar--month-length--31">
```

Reply:
done

Reply:
done